### ISSUE TITLE
Indented function call with multiple closure arguments can cause scope indent error

### ISSUE THREAD

**Issue by nkovacs** [squizlabs/PHP_CodeSniffer#1221]
```php
<?php

namespace test;

class Test
{
    public function foo()
    {
        $target->sentry->shouldReceive('capture')
            ->with(Mockery::on(function ($data) {
                return $data['message'] === 'sentinel';
            }), Mockery::on(function ($traces) {
                return true;
            }))->once();

        echo "foo";
    }
}
```

With PHP_CodeSniffer version 2.7.0 and the PSR2 standard this reports:

```
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 16 | ERROR | [x] Line indented incorrectly; expected at least 12 spaces,
    |       |     found 8 (Generic.WhiteSpace.ScopeIndent.Incorrect)
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

If there's only one function, the error isn't reported.

**Comment by gsherwood** [comment]
I've fixed this case now. Closures themselves don't cause problems, but when you've also indented the function call and put a second one in there without any sort of line breaks, it got confused because it didn't process the closing parenthesis after the `}` and so didn't reset the indent. It does now.

Thanks for the report.

**Comment by nkovacs** [comment]
Thanks!
